custom_components/pentair_cloud/switch.py

```
@@ -0,0 +1,201 @@
+        enabled_value = fields.get("d25")
+        if enabled_value is None:
+            return None
+        return enabled_value == "1"
+
+    def _handle_coordinator_update(self) -> None:
+        """Handle updated data from the coordinator."""
+        # Clear optimistic state when we get real data
+        self._optimistic_state = None
+        super()._handle_coordinator_update()
+
+    async def async_turn_on(self, **kwargs: Any) -> None:
```

Ruff (0.14.13)

Unused method argument: `kwargs`

(ARG002)